maven/arch.xml

```diff
@@ -286,6 +286,35 @@
         <code>address:port</code> or just port number to connect to.
         </p>
         </api>
+        <api category="stable" group="property" name="jpda.attach.trigger" type="export">
```

Comment:
This produces an anchor in `architecture-overview.html#property-jpda.attach.trigger` or something like that.